be/src/fs/hdfs/hdfs_fs_cache.cpp

```diff
@@ -82,6 +82,11 @@ static Status create_hdfs_fs_handle(const std::string& namenode, const std::shar
                               hedged_read_threshold_millis.data());
     }
 
+    hdfsBuilderConfSetStr(hdfs_builder, "fs.hdfs.impl", "com.starrocks.fs.CacheFileSystem");
+    hdfsBuilderConfSetStr(hdfs_builder, "fs.hdfs.impl.disable.cache", "true");
```

Comment:
can we put `"fs.hdfs.impl.disable.cache", "true"` logic into SRFileSystem

Reply:
No. Otherwise CacheFileSystem instance will be reused.  We don't expect that.